lib/locales/pl.ts

```diff
@@ -487,7 +487,7 @@ export const locale: Locale = {
     stanislaus_of_szczepanow_bishop_patron_of_poland: 'Św. Stanisława, biskupa i męczennika, głównego patrona Polski',
     stephen_i_of_hungary: 'Św. Stefana Węgierskiego',
     stephen_the_first_martyr: 'Św. Szczepana, pierwszego męczennika',
-    sunday_of_the_word_of_god: '{{week}}. Niedziela zwykła, lub Niedziela Słowa Bożego',
+    sunday_of_the_word_of_god: '{{week}} Niedziela zwykła, czyli Niedziela Słowa Bożego',
```

Comment:
@ciesiolka, just to confirm: `3 Niedziela zwykła, czyli Niedziela Słowa Bożego` is the form used in Polish, right? There is no `.` after the week number? Or do you use Roman Numerals (`III`) instead? :thinking:

Reply:
The period is unnecessary - I missed that one. Fix ongoing.
There's a rule which says that ordinals have appended period, but it can be omitted when it's clear from the context that the number is an ordinal. This is the case. Also, official liturgical books don't use periods in titles.

Reply:
Understood. If possible, use the format used by the official liturgical books (prefer books more recently published to the older ones), i.e. if there is no `.` after ordinal numbers, don’t use them in any l10n key (`.seasons` nor `.names`).